### PR TITLE
WIP: Pg17

### PR DESCRIPTION
**Pull request by Jo0**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -34,22 +32,26 @@ The engineering team regularly tests the extension on the following platforms:
...
 For Ubuntu you can follow the [postgres install instructions](https://www.postgresql.org/download/linux/ubuntu/) then run
+
 ```bash
 sudo apt-get install make gcc pkg-config clang postgresql-server-dev-14 libssl-dev
 ```
 
 Next you need [cargo-pgrx](https://github.com/tcdi/pgrx), which can be installed with
+
 ```bash
 cargo install --version '=0.10.2' --force cargo-pgrx
```
[the remark this answers is not in the document]
Comment (Jo0) [reply Readme.md]: Thanks for catching this

**Comment by Jo0** [comment]
Had to nuke my local env, but I was trying to sort this out.

https://github.com/timescale/timescaledb-toolkit/actions/runs/11450107123/job/31888427034#step:6:190

**Comment by Jo0** [comment]
> Had to nuke my local env, but I was trying to sort this out.
> 
> https://github.com/timescale/timescaledb-toolkit/actions/runs/11450107123/job/31888427034#step:6:190

So apparently this is in Rust 1.76?

https://github.com/reactive-graph/reactive-graph/issues/102

**Comment by Jo0** [comment]
Is there an argument against going to latest stable rust?

**Comment by Jo0** [comment]
Am I ultimately getting blocked by this failure? It's not building the new image with the target Rust 1.76 to be utilized in `CI / Tests` and `Clippy and rustfmt`?

https://github.com/timescale/timescaledb-toolkit/actions/runs/11464043831

Also how are you triggering manual re-runs @sebastianwebber? 😁

**Comment by Jo0** [comment]
Looking at past CI failures

```
   TimescaleDB only supports PostgreSQL 14, 15, 16 and 17
```

https://github.com/timescale/timescaledb-toolkit/actions/runs/11455829083/job/31872655924#step:5:37

I'm probably going to go ahead and remove the PG 12 and 13 test references.

**Comment by Jo0** [comment]
Recent commit fixes

```
- name: Build and install TimescaleDB
      if: ${{ (github.event_name == 'schedule' && github.event.schedule == '0 8 * * 1-4') || inputs.tsdb-commit != '' }}
      run: ./tools/install-timescaledb '${{ matrix.pgversion }}' '${{ inputs.tsdb-repo || 'https://github.com/timescale/timescaledb.git' }}' '${{ inputs.tsdb-commit == '' && 'main' || matrix.tsdb_commit || inputs.tsdb-commit }}'
```

from 

```
run: ./tools/install-timescaledb '${{ matrix.pgversion }}' '${{ matrix.tsdb_commit || inputs.tsdb-repo || 'https://github.com/timescale/timescaledb.git' }}' '${{ inputs.tsdb-commit == '' && 'main' || matrix.tsdb_commit || inputs.tsdb-commit }}'
```

also 

try to abide by the original sentiment of maintaining support for PG 13 in similar fashion to how we were continuing to test for PG 12

```
        include:
        # TimescaleDB as of 2.12.0 no longer supports PostgreSQL 12.
        # To allow us to do run CI against PostgreSQL 12, we therefore explicitly
        # remove all CI's, except if we are run against the latest supported TimescaleDB
        # version for PostgreSQL 12, which is 2.11.2
        - pgversion: 12
          container:
            image: debian-11-amd64
            os: debian
            version: "11" 
          tsdb_commit: 2.11.2
        # TimescaleDB as of 2.16.0 no longer supports PostgreSQL 13.
        # To allow us to do run CI against PostgreSQL 13, we therefore explicitly
        # remove all CI's, except if we are run against the latest supported TimescaleDB
        # version for PostgreSQL 13, which is 2.15.3
        - pgversion: 13
          container:
            image: debian-11-amd64
            os: debian
            version: "11" 
          tsdb_commit: 2.15.3
```

**Comment by Jo0** [comment]
I think at this point I'm poking at too much. I will wait for the next GH Action runs and set of feedback

**Comment by Jo0** [comment]
> Am I ultimately getting blocked by this missing `GH_TOKEN` failure? It's not building the new image with the target Rust 1.76 to be utilized in `CI / Tests` and `Clippy and rustfmt`?
> 
> https://github.com/timescale/timescaledb-toolkit/actions/runs/11464043831
> 

Been reviewing my changeset and looking through the workflows.

Getting the image built seems to be my blocker right now.

**Comment by philkra** [comment]
@Jo0 thank you for your contributions, the GitHub actions are failing as the PR comes from a fork. I'll merge this PR into a local branch and go from there.

**Pull request merged**